java/private/java_test_suite.bzl

```diff
@@ -80,7 +85,7 @@ def java_test_suite(
         package = package,
         define_library = _define_library,
         # Default to bazel's default test runner if we don't know what people want
-        define_test = _TEST_GENERATORS.get(runner, _define_junit4_test),
+        define_test = test_generators.get(runner, _define_junit4_test),
```

Comment:
Hm. Perhaps this should be:
```
test_generators.get(runner, _TEST_GENERATORS.get(runner, _define_junit4_test))
```
This way users have access to the default generators by just leaving them out of `test_generators`.

Reply:
Jeff pointed out offline that the default generators _are_ exported. See https://github.com/bazel-contrib/rules_jvm/blob/main/java/defs.bzl#L40-L41